### PR TITLE
Changed the creation of the update notification link

### PR DESCRIPTION
**Pull request by Faldon**
Instead of chaining the DOM manipulation commands, create an object from a concatenated string.
Solves #273

### CODE REVIEW

File: apps/updatenotification/js/notification.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ $(document).ready(function(){
 		version = oc_updateState.updateVersion,
 		docLink = oc_updateState.updateLink,
 		text = t('core', '{version} is available. Get more information on how to update.', {version: version}),
-		element = $('<a>').attr('href', docLink).attr('target','_blank').text(text);
+		element = $('<a href="'+docLink+'" target="_blank">'+text+'</a>');
```
Comment (LukasReschke) [inline apps/updatenotification/js/notification.js]: Please make sure to have text and docLink escaped in this cases then using escapeHTML.

**Comment by MorrisJobke** [comment]
Tested and works 👍 

Code looks good

**Comment by schiessle** [comment]
looks good :+1:

**Comment by schiessle** [comment]
@nickvergessen

**Comment by jancborchardt** [comment]
Great stuff @Faldon, will add you to the organization and the Javascript team. :)

**Comment by Faldon** [comment]
@jancborchardt With pleasure. I finally thought, it's time to contribute back. That's the point in OSS.

**Comment by jancborchardt** [comment]
@Faldon welcome to Nextcloud then! :) Very much looking forward to future contributions.

**Comment by LukasReschke** [comment]
🚀 🎉

**Pull request merged**